app/views/pulitzer/text_editors/_simple_text_editor.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id="toolbar">
```

Comment:
@gomayonqui, I wanted this to come from the parent app. That way it's very simple to control how the editor works.

The idea is that this html would appear as a hidden element on any page that renders pulizter views. Pulitzer can find it by a data attribute. That way pulitzer doesn't need to know anything about the text editor configuration.

You could leave this in as a default, but it should first try to find the editor html int he partent app by a data-attribute, like data-pulitzer-editor="jobs" or data-pulitzer-editor without a value for the default.